test/helpers/api-integration/v3/object-generators.js

```diff
@@ -13,8 +13,13 @@ import * as Tasks from '../../../../website/server/models/task';
 // parameter, such as the number of wolf eggs the user has,
 // , you can do so by passing in the full path as a string:
 // { 'items.eggs.Wolf': 10 }
-export async function generateUser (update = {}) {
-  let username = (Date.now() + generateUUID()).substring(0, 20);
+//
+// To manually set a username pass it as the second parameter.
+// If no password is passed or it is falsy then a username will
+// be auto-generated.
+// Example: generateUser({}, 'TestName') adds user with the 'TestName' username.
+export async function generateUser (update = {}, manualUsername = null) {
```

Comment:
thanks for this, could you make this a bit more generic?

So the second optional parameter would be an object, empty by default, that can store `{username, email, password}` to be used in place of the defaults value

thanks!

Reply:
Great suggestion!